src/scene_battle_rpg2k3.h

```diff
@@ -18,6 +18,11 @@
 #ifndef EP_SCENE_BATTLE_RPG2K3_H
 #define EP_SCENE_BATTLE_RPG2K3_H
 
+// FIXME: How many CBA move frames does RPG_RT actually use?
+#define CBA_NUM_MOVE_FRAMES_STEP 15
```

Comment:
Please use constexpr and follow the naming convention . See https://github.com/EasyRPG/Player/blob/831c1a42141252eaf2f84aa01634de7318d9160e/src/scene_battle.h#L65

(you can squash this commit)

Reply:
Fixed.